src/Avalonia.Layout/NonVirtualizingLayoutContext.cs

```diff
@@ -0,0 +1,11 @@
+﻿// This source file is adapted from the WinUI project.
+// (https://github.com/microsoft/microsoft-ui-xaml)
+//
+// Licensed to The Avalonia Project under MIT License, courtesy of The .NET Foundation.
+
+namespace Avalonia.Layout
+{
+    public abstract class NonVirtualizingLayoutContext : LayoutContext
```

Comment:
Documentation comment missing.

Reply:
Yeah, this is just a placeholder class for now, I'm working on adding the stuff necessary for non-virtualizing layouts in another PR. I can add a doc comment though.